workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
+Containers provide visibility into the health and performance of your containerized environments. This section does not include containers that are part of an orchestration tool (e.g. Kubernetes). 
 
-### CPU Usage
-This shows the percentage of CPU used. The CPU usage of a container refers to the percentage of the host system's CPU resources that are currently being used by the container's processes.
+### Processes
 
-### RAM Usage
-This shows the percentage of RAM used. The container RAM usage in percentage refers to the proportion of allocated memory resources that a container is currently using on the host system.
+Processes are managed within your containers to ensure your Infrastructure is effectively monitored, maintained, and optimized for performance, reliability, and operability.
 
-### Started Ago
-This indicates when the container was started. when a container is started, and records the time it was started and saves this information as metadata associated with the container.
+### Kubernetes
 
-Filters
--------
-We have many options for data filters. Like selecting any date and searching container by a container name, container id, container image, etc. Also in the sidebar section, we also have multiple check options available to filter the containers list data.
+Kubernetes is a container orchestration platform that automates the deployment, scaling, management, and operation of containerized applications. Effortlessly personalize your k8s data visualization experience.
 
-![](/images/infra-monitoring/c2.png)
+<Accordion title="Clusters">
+  A cluster is a collection of nodes that operate together as a single unit. These include control plane components and nodes that run the actual workloads.
+</Accordion>
 
-Container Details
------------------
-Click on any container to view more details, the tab view provides basic container information such as metrics in a graphical view, hostname, and container image. the info will always display the latest data. You can view resources metrics and logs related to that container in this detail view.
+<Accordion title="Nodes">
+    A node is a physical or virtual machine within a Kubernetes cluster where containers are deployed and run.
+</Accordion>
 
-![](/images/infra-monitoring/c3.png)
+<Accordion title="Services">
+    A service defines a particular set of pods and provides a consistent network endpoint to access them. A service can be used for load balancing, service discovery, and a layer of abstraction for connecting to applications within a cluster.
+</Accordion>
 
-### CPU Utilization (%)
-The container CPU percent refers to the percentage of CPU resources that a container is currently using on the host system. The container runs as a separate process on the host system and is allocated a certain amount of CPU resources based on its configuration and resources that are currently being used by the container's processes.
+<Accordion title="Workloads">
```

Comment:
This whole accordion section is awesome. Super explicit and yet descript. Nice work.